google-cloud-trace/lib/google/cloud/trace/faraday_middleware.rb

```diff
@@ -20,11 +20,36 @@ module Google
   module Cloud
     module Trace
       class FaradayMiddleware < Faraday::Middleware
+        ##
+        # # Trace FaradayMiddleware
+        #
+        # A faraday middleware that setup request/response labels for trace.
+        #
+        # ## Installing
+        #
+        # To use this middleware, simply install it in your middleware stack.
+        # Here is an example configuration enable the Trace middleware:
+        #
+        # ```ruby
+        # conn = Faraday.new(:url => 'http://sushi.com') do |faraday|
```

Comment:
To conform to the standard style for this repo, please use:
```ruby
conn = Faraday.new url: "http://example.com" do |faraday|
```

Reply:
Updated